lib/performance/EmittedAssetSizeLimitPlugin.js

```diff
@@ -138,17 +138,17 @@ EmittedAssetSizeLimitPlugin.prototype.apply = function(compiler) {
 		// if 1, then 2, then 3, 
 
 		if(assetsOverSizeLimit.length) {
-			warnings.push(new AssetsOverSizeLimitWarning(assetsOverSizeLimit, compilation));
-			warnings.push(new EntrypointsOverSizeLimitWarning(entrypointsOverSizeLimit, compilation));
+			warnings.push(new AssetsOverSizeLimitWarning(assetsOverSizeLimit, compilation, formatSize));
+			warnings.push(new EntrypointsOverSizeLimitWarning(entrypointsOverSizeLimit, compilation, formatSize));
 		} else {
 			if(entrypointsOverSizeLimit.legnth) {
-				warnings.push(new EntrypointsOverSizeLimitWarning(entrypointsOverSizeLimit, compilation));
+				warnings.push(new EntrypointsOverSizeLimitWarning(entrypointsOverSizeLimit, compilation, formatSize));
 			}
 		}
 
 		if(!hasAsyncChunks) {
 			warnings.push(new Error("EmittedAssetSizeWarning: NoAsyncChunks: " +
-				"You can limit the size of your bundles by using System.import() or require.ensure to lazy load some parts of your application.\n" + 
+				"You can limit the size of your bundles by using System.import() or require.ensure to lazy load some parts of your application.\n" +
 				"For more info visit https://webpack.github.io/docs/code-splitting.html"
 			));
```

Comment:
Oh wait nevermind, this feature will only land in webpack 2. So it would be either `System.import` and `require.ensure` as possible options. (Or `import()` for the future).